lib/class-sp-api.php

```diff
@@ -158,7 +158,7 @@ public function search( $query, $args = array() ) {
 	}
 
 	public function cluster_health() {
-		$health_uri = apply_filters( 'sp_cluster_health_uri', '/_cluster/health/' . $this->index );
+		$health_uri = apply_filters( 'sp_cluster_health_uri', '/_cluster/health/' . $this->index . '?wait_for_status=yellow&timeout=0.2s' );
```

Comment:
I thought about doing that, but it ultimately is being filtered. If someone wanted to change it, they could do so through this filter (`sp_cluster_health_uri`). I'll add a docblock highlighting that.